crates/polars-ops/src/series/ops/arg_min_max.rs

```diff
@@ -74,33 +74,50 @@ impl ArgAgg for Series {
 }
 
 pub(crate) fn arg_max_bool(ca: &BooleanChunked) -> Option<usize> {
-    if ca.is_empty() {
+    if ca.is_empty() || ca.null_count() == ca.len() {
         None
-    } else if ca.null_count() == ca.len() {
-        Some(0)
     }
     // don't check for any, that on itself is already an argmax search
     else if ca.null_count() == 0 && ca.chunks().len() == 1 {
         let arr = ca.downcast_iter().next().unwrap();
         let mask = arr.values();
         Some(first_set_bit(mask))
     } else {
+        let mut first_false_idx: Option<usize> = None;
         ca.into_iter()
-            .position(|opt_val| matches!(opt_val, Some(true)))
+            .enumerate()
+            .find_map(|(idx, val)| match val {
```

Comment:
The logic is: we first find `true`, and if it does not exist, we return the first `false` encountered. Is this what we want?

Reply:
@reswqa Yes, that's the correct behavior, although I think we can do a faster implementation more explicitly iterating over the bitmap. But that doesn't have to be in this PR.